homeassistant/components/lookin/climate.py

```diff
@@ -151,6 +152,28 @@ async def async_set_temperature(self, **kwargs: Any) -> None:
         if (temperature := kwargs.get(ATTR_TEMPERATURE)) is None:
             return
         self._climate.temp_celsius = int(temperature)
+        lookin_index = LOOKIN_HVAC_MODE_IDX_TO_HASS
+        if hvac_mode := kwargs.get(ATTR_HVAC_MODE):
```

Comment:
This usually isn't sent, but the service does support it https://www.home-assistant.io/integrations/climate/#service-climateset_temperature